asset/js/setup.js

```diff
@@ -1,7 +1,41 @@
 /* eslint-disable no-undef */
 
 Vue.use(VueStrap);
-const vm = new Vue({
-  el: '#app',
-});
-VueStrap.installEvents(vm);
+
+const { typeahead } = VueStrap.components;
+
+function setup() {
+  const vm = new Vue({
+    el: '#app',
+  });
+  VueStrap.installEvents(vm);
+}
+
+function setupWithSearch(siteData) {
+  const titleArray = jQuery.map(siteData.pages, object => object.title);
+  const routeArray = jQuery.map(siteData.pages, object => object.src);
+
+  const vm = new Vue({
+    el: '#app',
+    components: {
+      typeahead,
+    },
+    data() {
+      return {
+        searchData: titleArray,
+      };
+    },
+    methods: {
+      searchCallback(match) {
+        const index = titleArray.indexOf(match);
+        const route = routeArray[index];
+        window.location.pathname = route.replace('.md', '.html');
```

Comment:
The change in https://github.com/MarkBind/markbind-cli/pull/32#discussion_r170452815 wasn't made?

Reply:
It was. I changed it to replace '.md' with '.html' instead of just 'md'.